src/lib.rs

```diff
@@ -1353,6 +1353,23 @@ where
     }
 }
 
+impl<K, V> Clone for FlurryHashMap<K, V, RandomState>
+where
+    K: Sync + Send + Clone + Hash + Eq,
+    V: Sync + Send + Clone,
+{
+    fn clone(&self) -> Self {
+        let cloned_map = FlurryHashMap::with_capacity(self.count.load(Ordering::SeqCst));
+        {
+            let guard = epoch::pin();
+            for (k, v) in self.iter(&guard) {
```

Comment:
Let us add a note under "consistency" in the top-level documentation that `Clone` may not produce a "perfect" clone if the underlying map is being concurrently modified.

Reply:
That was one of my concerns. How could we do it? I guess some `RWLock`, but is it worth?

Reply:
Ah, no, I think we specifically _want_ `clone` to be racy. We just need to make sure to call it out in the documentation, just like the Java version does :)